browser/ui/webui/ai_chat/ai_chat_ui.cc

```diff
@@ -84,25 +85,33 @@ AIChatUI::AIChatUI(content::WebUI* web_ui)
   constexpr bool kIsMobile = false;
 #endif
 
-  untrusted_source->AddBoolean("isMobile", kIsMobile);
-  untrusted_source->AddBoolean("isHistoryEnabled",
-                               ai_chat::features::IsAIChatHistoryEnabled());
+  source->AddBoolean("isMobile", kIsMobile);
+  source->AddBoolean("isHistoryEnabled",
+                     ai_chat::features::IsAIChatHistoryEnabled());
 
-  untrusted_source->OverrideContentSecurityPolicy(
+  web_ui->AddRequestableScheme(content::kChromeUIUntrustedScheme);
+  source->OverrideContentSecurityPolicy(
       network::mojom::CSPDirectiveName::ScriptSrc,
-      "script-src 'self' chrome-untrusted://resources;");
-  untrusted_source->OverrideContentSecurityPolicy(
+      "script-src 'self' chrome://resources;");
+  source->OverrideContentSecurityPolicy(
       network::mojom::CSPDirectiveName::StyleSrc,
-      "style-src 'self' 'unsafe-inline' chrome-untrusted://resources;");
-  untrusted_source->OverrideContentSecurityPolicy(
+      "style-src 'self' 'unsafe-inline' chrome://resources;");
+  source->OverrideContentSecurityPolicy(
       network::mojom::CSPDirectiveName::ImgSrc,
-      "img-src 'self' blob: chrome-untrusted://resources;");
-  untrusted_source->OverrideContentSecurityPolicy(
+      "img-src 'self' blob: chrome://resources chrome://favicon2;");
+  source->OverrideContentSecurityPolicy(
       network::mojom::CSPDirectiveName::FontSrc,
-      "font-src 'self' data: chrome-untrusted://resources;");
+      "font-src 'self' data: chrome://resources;");
```

Comment:
do we need `data:`?

Reply:
I believe some of the fonts get base64 encoded by webpack instead of a separate file. I could be wrong. Possibly this one can be removed now that we embed our fonts without webpack compile at chrome://resources, and we don't have an icon font.

Reply:
I removed `data:`